### PR TITLE
CRAW-200 Create endpoint for websites

### PR DESCRIPTION
**Pull request by ross-hike**
https://hikeseo.atlassian.net/browse/CRAW-200

### CODE REVIEW

File: database/migrations/2024_10_17_211015_create_websites_table.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('websites', function (Blueprint $table) {
```
Comment (mante-hike) [inline database/migrations/2024_10_17_211015_create_websites_table.php:14]: Could we add a safety check for `Schema::hasTable` here mate?

File: tests/Feature/Http/Controllers/Websites/CreateControllerTest.php [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+        $this->postJson(route('api.websites.create'), [])
+            ->assertUnauthorized();
+    }
+
+    #[DataProvider('validatesInputDataProvider')]
+    public function test_validates_input(array $input, array $expectedErrors): void
+    {
+        Website::factory()->create([
+            'url' => 'https://example.com',
+        ]);
```
Comment (mante-hike) [inline tests/Feature/Http/Controllers/Websites/CreateControllerTest.php:31]:
```suggestion
        ]);
        
```

File: database/migrations/2024_10_17_211015_create_websites_table.php (hunk 1 above)
Comment (ross-hike) [reply database/migrations/2024_10_17_211015_create_websites_table.php:18]: Done

File: tests/Feature/Http/Controllers/Websites/CreateControllerTest.php (hunk 2 above)
Comment (ross-hike) [reply tests/Feature/Http/Controllers/Websites/CreateControllerTest.php:31]: Done

**Review by Dan-Hike — APPROVED** [review]

**Pull request merged**